docs/section9/Rule6-ja.md

```diff
@@ -0,0 +1,42 @@
+
+# Lighting - Rule 6-ja
+
+**Rule ID:** 6-ja  
+**Rule Description:** The total building interior lighting power shall not exceed the interior lighting power allowance determined using either Table G3.7 or G3.8.  
+**Appendix G Section:** Lighting  
+**Appendix G Section Reference:**  
+
+- Table G3.7, Performance Rating Method Lighting Power Density Allowances and Occupancy Sensor Reductions Using the Space-by-Space Method
+- Table G3.8, Performance Rating Method Lighting Power Densities Using the Building Area Method  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** Table G3.7 and Table G3.8  
+## Rule Logic: 
+
+- Get the building area lighting area type of the building segment in the Proposed model: ```building_light_area_type = building_segment.lighting_building_area_type``` (Note XC, assuming this field lists values from T-9.5.1 or T-G3.8, for "Building Area Method", or "None" if the building goes for "Space-by-Space Method")
+
+- Get thermal_block from building segment: ```thermal_block in building_segment.thermal_blocks:```
+
+- Get thermal_zone from thermal block: ```thermal_zone in thermal_block.zones:```
+
+- Get space from thermal zone: ```space in thermal_zone.spaces:```
+
+- Get floor_area from space: ```space_floor_area = space.floor_area```
+
+  - If building_light_area_type is in Table G3.8 then add floor_area to the total building floor area: ```total_building_area.append(space_floor_area)```
```

Comment:
@JasonGlazer and @xingcx a building is likely to not have spaces defined if they use the building area method. I see that thermal zones do not have an area data element, is the assumption that there will be a one-is-to-one relationship between thermal zones and spaces and the space data group would have the entire zone area?

Reply:
Here the 'total_building_area' is just the total_building_segment_area. A building might have more than one building segment with a different lighting_building_area_type. we should repeat the entire process for each building segment in the building.

Reply:
> @JasonGlazer and @xingcx - for the BAM, i believe the assumption that there will be a one-is-to-one relationship between thermal zones and spaces and the space data group would have the entire zone area. Is that correct?

I thought for BAM one type will be applied to the whole building. But we will need to sum up the areas of all spaces under one thermal zone, as you mentioned one thermal zone might have multiple spaces.

Reply:
I think the building area method applies to the building segment so it should be  added up to that aggregation level.

Reply:
Actually, reading it further, I think the building area method is really at the building level, See 9.5.1 (d)

Reply:
@JasonGlazer you're right. So we will have to calculate the proposed lighting power and the total lighting power allowance for each building segment, aggregate it up to the building level and determine rule outcome at the building level.

Reply:
I added a flag for each building segment to decide if it's using BAM. And the allowance is determined for each building segment, then add up at the building level. Updated in #22 .

Reply:
We don't need to know if building segment used BAM or space-by-space. The allowances must be calculated using both methods (G3.7 and G3.8), and project complies if it meets EITHER of the requirements.

Reply:
@mkarpman we do have an indicator if we are using BAM or space-by-space. Are you suggesting that we don't use that? If so why?

Reply:
@JasonGlazer I believe this indicator it's irrelevant for establishing PRM compliance. Baseline is ALWAYS space-by-space. Proposed is ALWAYS as specified if lighting design exists or for existing lighting, and ALWAYS BAM if there is no lighting design. Mandatory requirement is to meet G3.7 OR G3.8, so RCT would check it both ways and if it passes either of the two then project complies. So why do we need to have BAM vs space-by-space indicator?

Reply:
@mkarpman the approach you are saying is valid but I'm not sure it is better. The data model indicates it if the BAM or not by the presence of building_segment.lighting_building_area_type, if it is None then it needs to be space by space.

Reply:
@JasonGlazer Based on my understanding of the PRM rules described above, it's always prescribed whether space-by-space or BAM should be used. So, if  building_segment.lighting_building_area_type is blank but lighting is not specified, we don't have enough info to check whether baseline is established correctly. However, since we may not know whether lighting is actually specified or not (as it will always be modeled in the proposed design based on the PRM rules), I think RCT checks may always verify if baseline is set correctly based on space-by-space method, and if it passes, provide warning message for reviewer to verify that lighting is fully specified. If it fails, then there is either a mistake or lighting is not specified and BAM was used. If the latter, than lighting for the proposed design should be based on space-by-space allowances in Table 9.

Reply:
Addendum af actually revised the language for the this part. When the design does not exist or is not submitted and uses Building Area Method, the baseline will use BAM as well.
 https://www.ashrae.org/file%20library/technical%20resources/standards%20and%20guidelines/standards%20addenda/90_1_2019_af_bc_cd_db_20201116.pdf

For Proposed Building

> c. Where lighting neither exists nor is submitted with design documents, lighting shall comply with but not exceed the requirements of Section 9. **Where space types are known, lighting power shall be determined in accordance with the Space-by-Space Method. Where space types are not known, lighting Lighting power shall be determined in accordance with the Building Area Method**.

For Baseline

> Interior lighting power in the baseline building design shall be determined using the values in Table G3.7. **However, where lighting neither exists nor is submitted with design documents, and the proposed design lighting power is determined in accordance with the Building Area Method, the baseline building design lighting power shall be determined in accordance with Table G3.8.** Where retail display lighting is included in the proposed building design in accordance with Section 9.6.2(b), the baseline building design retail display lighting additional power shall be equal to the limits established by Section 9.6.2(b) or same as proposed, which ever less.

I made some minor changes to #22 , and will update Rule 6-o accordingly.